### ISSUE TITLE
Bug, The extension doesn't work at all, stuck on Loading Ren'py static data

### ISSUE THREAD

**Issue by DeltaGe** [renpy/vscode-language-renpy#389]
### Extension version

2.3.6

### Issue description

Howdy there. The extension would be really nice if it worked, and it does work on the question game included in the original renpy distribution. I tried installing this one extension in VScode and running development with it, multiple PCs, multiple windows versions, multiple reinstalls of VScode, multiple things tried, every single time it fails to load, it is either stuck on "loading ren'py static data", or it simply hangs the entire VScode environment. It disables intellisense entirely for all other extensions. 
I would really love to help or do something to bring it back, because it would theiretically have better support than the other extension, better color coding, but as it right now does not load intellisense...
At any rate, it does not seem to work with any project I have, it does not load. I am running a workstation, 
7950x, 64GB RAM, 4090, enough hardware to run it even if it tried to read the entire code of the entire project. It does not spike CPU or RAM usage in Task Manager. 
I can provide logs if you can tell me how to do that, I would really like to use it. 

### Scope inspector screenshots

--

### Code

--

**Comment by EtchJetty** [comment]
Having this same issue, too. I keep having to disable the extension whenever opening an .rpy file over a certain size.

**Comment by duckdoom4** [comment]
We are aware of the issue, unfortunately I've not been able to reproduce it locally. The issue occurs on specific renpy files. The size is irrelevant.

It's due to the custom parser getting stuck in a loop and unfortunately we can't run the code on a separate thread to mitigate the side effects of it getting stuck.

To help us solve the issue I need a minimal repro example, try to pinpoint the exact file that is causing the slowdown and stip it down to something you can share

**Comment by duckdoom4** [comment]
As for the OP, this has also happened to people every now and then. Unfortunately we have never been able to figure out what is going on there.

There must be a specific thing that you set-up differently than we (the development team) have. We have some options to display logs, so those would be very helpful. Other then that, could you tell us step by step how you set-up everything?

Eg. are you using the visual studio code version provided by renpy or a local install? Is it a new or old project? Did you try using the renpy sample project?
Is your setup crossing drive letters? Eg. C: drive contains visual studio, but D: drive contains your project.

At this point I'm looking for any kind of clue cus we're kinda lost on this one

**Comment by burnerid404** [comment]
@duckdoom4, I have found a way to reproduce it.

I used a game available on [itch.io](https://itch.io/) and extracted all of the files from `scripts.rpa` using [RPA Extractor Online](https://grviewer.com/tab/rpa-extractor-online), then deleted `scripts.rpa` from the `game` folder, so I could make edits and they would appear in-game.

When I opened the main script file, with most of the story and dialogue, the error occurred.

However, I found out two things by chance that may help.

1. The error only happens if you open VS Code and a session with that file opened is restored. It does not happen if you open VS Code and then open the file.

2. And it only happens if the file is in the `game` folder, with all other scripts and files.

Meaning, the error does not happen if I isolate the file, and then open VS Code and it restores a session with that file opened.

If need be, I'll tell you which game I used. in private.
